apps/package.json

```diff
@@ -83,7 +83,7 @@
     "ejs": "0.8.4",
     "ejs-compiled-loader": "2.1.1",
     "ejs-loader": "0.2.1",
-    "enzyme": "2.4.1",
+    "enzyme": "^2.9.1",
```

Comment:
I believe you're right that it should. I still don't have a full understanding/trust of yarn.

I'd just as soon also have our package.json point at specific versions. Otherwise, you can presumably get into situations where your package.json says something like `^2.9.1`, yarn has pinned to `2.9.8` and you become confused by the discrepancy. It also means that if someone uses `npm install` instead of `yarn install` they end up with the same version.